src/EnergyPlus/MixedAir.cc

```diff
@@ -4233,12 +4253,48 @@ namespace MixedAir {
 		// accurate result using a full mass, enthalpy and moisture balance and iteration.
 		if ( OutAirSignal > OutAirMinFrac && OutAirSignal < 1.0 && this->MixMassFlow > VerySmallMassFlow && this->ControllerType_Num == ControllerOutsideAir && ! AirLoopNightVent ) {
 			Par( 1 ) = this->MixNode;
-			Par( 2 ) = this->RetNode;
-			Par( 3 ) = this->InletNode;
+			Par( 2 ) = this->RelNode;
+			Par( 3 ) = this->OANode;
 			Par( 4 ) = this->MixMassFlow;
-			SolveRegulaFalsi( Acc, MaxIte, SolFla, OASignal, MixedAirControlTempResidual, OutAirMinFrac, 1.0, Par );
-			if ( SolFla < 0 ) {
-				OASignal = OutAirSignal;
+			Par( 5 ) = 0.0;
+			if ( FirstHVACIteration ) Par( 5 ) = 1.0;
+			Par( 6 ) = double( AirLoopNum );
+
+			Node( this->OANode ).MassFlowRate = OutAirMinFrac * Node( this->MixNode ).MassFlowRate;
+			Node( this->RelNode ).MassFlowRate = OutAirMinFrac * Node( this->MixNode ).MassFlowRate;
+			for( CompNum = 1; CompNum <= OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).NumComponents; ++CompNum ) {
+				CompType = OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).ComponentType( CompNum );
+				CompName = OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).ComponentName( CompNum );
+				SimOAComponent( CompType, CompName, OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).ComponentType_Num( CompNum ), FirstHVACIteration, OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).ComponentIndex( CompNum ), AirLoopNum, Sim, AirLoopControlInfo( AirLoopNum ).OASysNum, OAHeatCoil, OACoolCoil, OAHX );
+			}
+			lowFlowResiduum = Node( this->MixNode ).TempSetPoint - Node( this->MixNode ).Temp;
+			lowFlowOA = Node( this->OANode ).MassFlowRate;
+			lowMixMassFlow = Node( this->MixNode ).MassFlowRate;
+
+			Node( this->OANode ).MassFlowRate = Node( this->MixNode ).MassFlowRate;
+			Node( this->RelNode ).MassFlowRate = Node( this->MixNode ).MassFlowRate;
+			OAHeatCoil = false;
+			OACoolCoil = false;
+			OAHX = false;
+			Sim = true;
+			for( CompNum = 1; CompNum <= OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).NumComponents; ++CompNum ) {
+				CompType = OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).ComponentType( CompNum );
+				CompName = OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).ComponentName( CompNum );
+				SimOAComponent( CompType, CompName, OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).ComponentType_Num( CompNum ), FirstHVACIteration, OutsideAirSys( AirLoopControlInfo( AirLoopNum ).OASysNum ).ComponentIndex( CompNum ), AirLoopNum, Sim, AirLoopControlInfo( AirLoopNum ).OASysNum, OAHeatCoil, OACoolCoil, OAHX );
+			}
+			highFlowResiduum = Node( this->MixNode ).TempSetPoint - Node( this->MixNode ).Temp;
+			highFlowOA = Node( this->OANode ).MassFlowRate;
+			highMixMassFlow = Node( this->MixNode ).MassFlowRate;
+
```

Comment:
Once both results are known, a choice can be made as to which solution is used...the low OA flow, the high OA flow, or iterate on a solution.